src/Microsoft.Health.SqlServer/ManagedIdentitySqlConnection.cs

```diff
@@ -0,0 +1,67 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.SqlServer.Configs;
+
+namespace Microsoft.Health.SqlServer
+{
+    public class ManagedIdentitySqlConnection : ISqlConnectionFactory
```

Comment:
Should this and DefaultSqlConnection ends with Factory?

Reply:
Yeah, I will update.